openslides/motions/static/templates/motions/motion-list.html

```diff
@@ -223,7 +223,9 @@ <h1 translate>Motions</h1>
                     ng-options="user.id as user.full_name for user in users"
                     search-contains="true"
                     id="selectSubmitter"
-                    class="form-control">
+                    class="form-control"
+                    data-placeholder-text-multiple="'No submitter selected' | translate"
```

Comment:
Please use consistently strings for data-placeholder-text, e.g. "Select some submitters/supporters/tags".